models/RwdImage.php

```diff
@@ -156,6 +185,10 @@ public function img( $size, $attributes = array() ) {
 			return '';
 		}
 
+		if ( $this->is_svg_image() ) {
+			return $this->get_svg_image( $this->attachment->ID );
+		}
```

Comment:
you should just allow to print this image in configured image sizes, you should not terminate method execution